src/shared/nt-results/ResultsPage.react.js

```diff
@@ -64,13 +65,11 @@ class ResultsPage extends React.Component {
   render () {
     return (
       <div className={styles.nt}>
-        <div className={styles.nt__sidebar}>
-          {this.renderLink('setting')}
-          Sessions
+        <div className={styles.nt__nav}>
+          {this.renderLink('setting', 'cog')}
           {this.props.sessionTypes.map(sessionType => (
-            this.renderLink(url.parse(sessionType.url).path, `/${sessionType._id}`)
+            this.renderLink(url.parse(sessionType.url).path, '', `/${sessionType._id}`)
```

Comment:
if more than one argument of a method can be empty or null, you should pass arguments with an object type.

For example.
```javascript 
renderLink = (content, { iconName, path = '' }) => (
    <NavLink
      key={path}
      to={`${this.props.match.url}${path}`}
      activeClassName={styles.nt__activeLink}
      exact
    >
      <div className={styles.nt__link}>
        { iconName && <i className={`fa fa-${iconName}`} aria-hidden='true' />}
        <span>{content}</span>
      </div>
    </NavLink>
  )
....
...
...
this.renderLink('setting', { iconName: 'cog' })
....
this.renderLink(url.parse(sessionType.url).path, { path: `/${sessionType._id}` })
```

VSCode also have a guide when calling a method.
![image](https://cloud.githubusercontent.com/assets/5950717/25351916/5e7673f6-2954-11e7-8e41-2ec0498c5c67.png)